src/connection-monitor-preview/azext_connection_monitor_preview/tests/latest/test_connection_monitor_preview.py

```diff
@@ -0,0 +1,340 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.util import CLIError
+
+from azure_devtools.scenario_tests import AllowLargeResponse
+from azure.cli.testsdk import (ScenarioTest, ResourceGroupPreparer)
+
+
+class ConnectionMonitorPreviewScenarioTest(ScenarioTest):
+
+    def _prepare_vm(self, resource_group, vm):
+        vm_create_cmd_tpl = 'vm create -g {rg} --name {vm} ' \
+                            '--image UbuntuLTS ' \
+                            '--authentication-type password ' \
+                            '--admin-username deploy ' \
+                            '--admin-password PassPass10!) ' \
+                            '--nsg {vm} '
+        vm_info = self.cmd(vm_create_cmd_tpl.format(rg=resource_group, vm=vm)).get_output_in_json()
+
+        vm_enable_ext_tpl = 'vm extension set -g {rg} ' \
+                            '--vm-name {vm} ' \
+                            '--name NetworkWatcherAgentLinux ' \
+                            '--publisher Microsoft.Azure.NetworkWatcher '
+        self.cmd(vm_enable_ext_tpl.format(rg=resource_group, vm=vm))
+
+        return vm_info
+
+    def _create_default_connection_monitor_v2(self,
+                                              location,
+                                              connection_monitor,
+                                              source_name,
+                                              source_resource_id):
+        cmd_tpl = 'network watcher connection-monitor create ' \
+                  '--location {location} ' \
+                  '--name {cmv2} ' \
+                  '--endpoint-source-name {src_resource_name} ' \
+                  '--endpoint-source-resource-id {src_resource_id} ' \
+                  '--endpoint-dest-name bing ' \
+                  '--endpoint-dest-address bing.com ' \
+                  '--test-config-name DefaultTestConfig ' \
+                  '--protocol Tcp ' \
+                  '--tcp-port 2048 '
+        cmd = cmd_tpl.format(cmv2=connection_monitor,
+                             location=location,
+                             src_resource_name=source_name,
+                             src_resource_id=source_resource_id)
+
+        # connection monitor V2 is in preview stage
```

Comment:
Hello. Is this by purpose. should it success?

Reply:
As talked at Teams,  believe it could be some transient issue of conflict error with ARM/LA.

Reply:
Change default location to eastus in test